modules/security/data.tf

```diff
@@ -0,0 +1,62 @@
+data "aws_iam_policy_document" "cuddle_serverless_assume_role" {
+  statement {
+    effect = "Allow"
+    principals {
+      type = "Service"
+      identifiers = [ 
+        "lambda.amazonaws.com", 
+        "states.amazonaws.com",
+        "apigateway.amazonaws.com",
+         ]
+    }
+    actions = [ "sts:AssumeRole" ]
+
+  }
+}
+data "aws_iam_policy_document" "cuddle_serverless_lambda" {
+
+  statement {
+    effect = "Allow"
+    actions = ["ses:*"]
+    resources = ["*"]
+  }
+  statement {
+    effect = "Allow"
+    actions = ["sns:*"]
+    resources = ["*"]
+  }
+  statement {
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "logs:DescribeLogGroups",
+      "logs:DescribeLogStreams",
+      ]
+    resources = ["*"]
+  }
+  statement {
+      actions = [ "states:*" ]
+      effect = "Allow"
+      resources = [ "*" ]
+    }
+}
+data "aws_iam_policy_document" "cuddle_serverless_sfn" {
+    statement {
+      actions = [ "lambda:*" ]
+      effect = "Allow"
+      resources = [ "*" ]
```

Comment:
use this for test, it's okay, but do not push it
try to get used to restricting access to resources, this will allow you to know each resource what does it need as permission to correctly execute